etesync_server/wsgi.py

```diff
@@ -10,7 +10,24 @@
 import os
 
 from django.core.wsgi import get_wsgi_application
+from django.conf import settings
```

Comment:
I guess this line is a mistake and should be removed?

Reply:
You are right. I removed it.